tests/data/snapshots/test_browse/admin/_language0_.context.json

```diff
@@ -9,11 +9,11 @@
         "lastaction": {
           "displayname": "Member",
           "email": "a4fae232e2bfebd9f4dc8d7cb6caecb2",
-          "mtime": 1578661829,
-          "source": "Translated Source /language0/project0/subdir0/sto\u2026",
+          "mtime": 1591122676,
+          "source": "Translated Source /language0/project0/store0.po 2\u2026",
```

Comment:
Here and below, I'm concerned about these snapshot changes to the source string. They don't seem to be related to the changes this PR is supposed to introduce. We need to understand why this property is reported differently now.

Reply:
Don't know why it happens. Tests wouldn't pass if I kept the old snapshots. Might be because of my environment, but before the changes I introduced the tests would run smoothly.

Reply:
The thing is that when I run the tests without generating a new dump (not snapshots) I get this error on all tests when setting up the fixtures.
![Screen Shot 2020-07-09 at 1 03 09 PM](https://user-images.githubusercontent.com/16262338/87069185-c03aba80-c1e4-11ea-95be-6babc5c3ef9a.png)
Then, when I generate a new dump I can get past the fixtures setup but other errors show up. 

When I called the submission procedure outside the save method, generating a new dump and snapshots would fix all tests. However, there's that weird modification of source strings.

So the problem here would be that the original `data_dump.json` is having troubles when initialized on MySQL. Apparently because of uniqueness constraints. On the other hand when I create a new dump some of this problems dissapear.

Reply:
There was no reason to modify snapshots, fixed.

Reply:
@iafan It turns out I could not get over the uniqueness constraints colliding from the dump without generating a new dump, so I uploaded a new one. It contains some changes on ids, timestamps and ordering. 

The new dump would not pass all tests without regenerating snapshots, so I did regenerate them. The new snapshots have some weird source changes as you pointed out. I've looked back at commit history and found [a commit](https://github.com/evernote/zing/commit/81f4b9391b4d4af896a98b7f51024819b5093e9e) that had a similar impact on snapshots. The last modification of urls to be tested occurs [here]( https://github.com/evernote/zing/commit/107746b7f1e815c587c18e5ccbb7b8513c297a01), so I discard the source modifications are due to different urls being tested and just having out of date snapshots.